content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -19,17 +19,44 @@ We are continuously testing the Data Proxy with Cloudflare Workers. [Open an iss
 
 </TopBlock>
 
-
 ## Prerequisites
 
-- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) database with an accessible URL
-  - e.g. `mongodb+srv://<user>:<password>@cluster.mongodb.net`.
-- Free [Cloudflare Worker](https://workers.cloudflare.com/) account.
-- Free [GitHub](https://github.com/) account.
-- Node.js & NPM installed.
-- Git installed.
+- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) account
+- Free [Cloudflare Workers](https://workers.cloudflare.com/) account
+- Free [GitHub](https://github.com/) account
+- Node.js & NPM installed
+- Git installed
+
+## 1. Get a MongoDB Atlas connection string
+
+After you create a MongoDB Atlas database cluster, you can get its connection string. By default, that connection string does not include a database name.
+
+When you import your project in the Prisma Data Platform at [step 8](#8-import-your-project-into-the-prisma-data-platform), you must provide a connection string that includes a database name.
+
+However, you do not need to manually create the database in MongoDB Atlas, even if it does not yet exist. The project creation workflow in the Prisma Data Platform will create the database.
+
+1. [Create a free shared cluster](https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/) in MongoDB Atlas.
+2. On the _Security Quickstart_ screen, complete the initial security configuration.  
+   1. Add a new database user. 
+   2. Click **Add My Current IP Address** so that you can connect to the MongoDB shared cluster from your current system.
+   3. Click **Finish and Close**.
+3. [Get the database cluster connection string](https://www.mongodb.com/docs/guides/atlas/connection-string/).
+4. Paste the connection string in a text editor and add the collection name `Log`.
+   
+   The code sample below shows a connection string, that is originally provided by MongoDB Atlas, with the database name now added.
+
+   ```diff
+   # The first line is a default-style MongoDB Atlas connection string.
+   # The second line includes the inserted `Log` database name.
+   - mongodb+srv://<user>:<password>@cluster.mongodb.net/?retryWrites=true&w=majority
+   + mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority
+   ```
 
-## 1. Set up your Application
+You now have a complete connection string. You will paste the connection string when you create your project in [8. Import your project into the Prisma Data Platform](#8-import-your-project-into-the-prisma-data-platform).
+
+For security reasons, do not keep the connection string in the text editor or save it in a text file.
+
+## 2. Set up your application
```

Comment:
```suggestion
## 2. Set up your Application
```
Are we using a specific style guide/sentence vs title case? Seems kind of inconsistent when we capitalise things in the titles. For example step 8 capitalises project.

Reply:
I appreciate you noticing this @eemmiillyy. I didn't pay much attention to these but will get them to all stick to sentence-style capitalization.